c2corg_api/scripts/migration/documents/routes.py

```diff
@@ -80,6 +80,10 @@ def get_document(self, document_in, version):
             # (in v6 activities are required)
             activities = ['skitouring']
 
+        climbing_outdoor_type = None
+        if 'rock_climbing' in activities and 'hiking' not in activities:
+            climbing_outdoor_type = 'multi'
```

Comment:
No, we talk about a route field : 
https://github.com/c2corg/v6_api/blob/master/c2corg_api/models/route.py#L147-L148